doc/user/INTERACTIONS.md

```diff
@@ -42,8 +42,10 @@ Other options can be toggled directly by pressing the following hotkeys:
 * `Z`: the display of the FPS counter.
 * `R`: raytracing.
 * `D`: the denoiser when raytracing.
-* `U`: background blur when using a HDRi.
+* `U`: background blur, useful with an HDRI.
 * `K`: trackball interaction mode.
+* `F`: HDRI ambient lighting.
+* `J`: the display of the HDRI skybox.
```

Comment:
is there any logic behind choosing `F` and `J` as shortcut?

Reply:
Before this change, only W,F and J were available. I just picked randomly. BTW we have an issue to rework interactions and all the bindings (#443 )